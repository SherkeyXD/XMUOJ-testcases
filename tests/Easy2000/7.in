cav
okf
pnj
uzw
mld
ldu
tkn
tdk
qbx
vnj
hbm
bjz
xdp
cwr
rcv
cro
ppu
ync
gwt
yby
lcp
mro
spo
klr
ydj
ols
rdc
yij
hhu
rht
njm
ffb
gtn
pft
ass
cui
ulm
dks
abb
ued
bym
ibc
pxp
mvb
xkz
xfg
udq
txo
fhv
nrw
ldr
qzx
kce
gtb
icm
rpu
icm
nko
efi
wxk
hhw
krl
uve
ogp
tsh
tbt
mik
frq
zwm
tnh
pji
ejj
wmy
vvt