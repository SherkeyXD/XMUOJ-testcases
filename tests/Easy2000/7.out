a c v
f k o
j n p
u w z
d l m
d l u
k n t
d k t
b q x
j n v
b h m
b j z
d p x
c r w
c r v
c o r
p p u
c n y
g t w
b y y
c l p
m o r
o p s
k l r
d j y
l o s
c d r
i j y
h h u
h r t
j m n
b f f
g n t
f p t
a s s
c i u
l m u
d k s
a b b
d e u
b m y
b c i
p p x
b m v
k x z
f g x
d q u
o t x
f h v
n r w
d l r
q x z
c e k
b g t
c i m
p r u
c i m
k n o
e f i
k w x
h h w
k l r
e u v
g o p
h s t
b t t
i k m
f q r
m w z
h n t
i j p
e j j
m w y
t v v
